Sources/MessagingInApp/Extensions/GistExtensions.swift

```diff
@@ -0,0 +1,9 @@
+import Foundation
+import Gist
+
+internal extension Message {
+    // Used for getting details about the Message object for sending to logs.
+    var describeForLogs: String {
+        "id: \(messageId), queueId: \(queueId ?? "none"), campaignId: \(gistProperties.campaignId ?? "none")"
```

Comment:
since this is part of our sdk i'd rename `campaignId` to `delivery_id or cio_delivery_id`